sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/JmsIOTest.java

```diff
@@ -98,6 +106,8 @@ public void startBroker() throws Exception {
 
     // create JMS connection factory
     connectionFactory = new ActiveMQConnectionFactory(BROKER_URL);
+    connectionFactoryWithoutPrefetch =
+        new ActiveMQConnectionFactory(BROKER_URL + "?jms.prefetchPolicy.all=0");
```

Comment:
Explain why you're configuring the prefetch policy in this way? Does client code need to also do this in order to get the properties that the test verifies?